docker-compose.yaml

```diff
@@ -6,6 +6,13 @@ services:
     ports:
       - '8001:8001'
     network_mode: bridge
+  matching-service:
+    build:
+      context: ./matching-service
+      dockerfile: Dockerfile.public
```

Comment:
We should rename this to Dockerfile and remove the ECR Dockerfiles since we have fully migrated to Docker Hub's Container Registry.

Reply:
This will be fixed in a future PR.